web-server/v0.4/public/index.html

```diff
@@ -3,21 +3,24 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
-    <title>Pbench Dashboard</title>
+    <title>PBench Dashboard</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="apple-touch-icon" href="apple-touch-icon.png">
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/antd/2.12.2/antd.min.css">
     <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.12.0/css/patternfly.min.css">
     <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.12.0/css/patternfly-additions.min.css">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/ant-design-pro/1.3.0/ant-design-pro.min.css">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/antd/2.12.2/antd.min.css">
```

Comment:
Can we consider pulling these four `.css` files locally during install and reference within the tree instead of remotely?